src/olympia/reviews/tests/test_views.py

```diff
@@ -1943,7 +1943,10 @@ def test_flag_logged_in(self):
         response = self.client.post(
             self.url, data={'flag': 'review_flag_reason_spam'})
         assert response.status_code == 202
-        assert response.content == ''
+        data = json.loads(response.content)
+        assert data == {
+            'msg': 'Thanks; this review has been flagged for editor approval.'
```

Comment:
Ah, this reminds me that I should display a message in the UI explaining what happens next.